lib/rust/flexer/src/group.rs

```diff
@@ -129,25 +136,18 @@ pub struct Identifier {
     val:usize
 }
 
-impl Identifier {
-    /// Creates a new identifier.
-    pub fn new(val:usize) -> Identifier {
-        Identifier{val}
-    }
-}
-
 
 // === Trait Impls ===
 
 impl From<usize> for Identifier {
     fn from(id:usize) -> Self {
-        Identifier::new(id)
+        Identifier {val:id}
```

Comment:
What do you mean by a "tuple-like" wrapper?

Reply:
I meant defining `Identifier` as `pub struct Identifier(usize)` — the advantage of record-like structure is that it introduces named fields and forces user code to use these names. But if the name is meaningless (and tbh I consider `value` to be such), this just causes clutter. 
With tuple-like struct we'd have just `Identifier(id)` rather than `Identifier {value:id}`.

Reply:
Got it. This should now be resolved.